### PR TITLE
add new user settings table

### PR DESCRIPTION
**Pull request by botany-lead**
add new user settings table

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -45,3 +45,5 @@ use new logo
 update auth flow
 
 more documentation
+
+add new user settings table
```
Comment (botany-demo) [inline readme.md:49]: thanks for adding a trigger to auto-update the `updated_at` field @botany-lead 💯

Comment (botany-lead) [reply readme.md:49]: no problem 👍

**Review by botany-demo — APPROVED** [review]
looks great!

**Comment by botany-demo** [comment]
this should be ready @botany-lead

**Comment by botany-lead** [comment]
ok merging now

**Pull request merged**